### PR TITLE
Put auditing and rollbar into the feature flags system

### PR DESCRIPTION
**Pull request by sungo**

### CODE REVIEW

File: Conch/conch.conf.dist [hunk 1]
```diff
@@ -18,7 +14,14 @@
     # rollbar_environment => 'staging',
 
 	features => {
+		'audit' => 0,
 		#'orc' => 1,
+		'rollbar' => 1,
```
Comment (lseppala) [inline Conch/conch.conf.dist]: Let's make the rollbar 0 in .dist, and uncomment orc and set as default off (or on, whichever you want).

Comment (lseppala) [reply Conch/conch.conf.dist]: Oh, also mind running perltidy on this file. Mix of spaces and tabs

Comment (sungo) [reply Conch/conch.conf.dist]: Will do.

Comment (sungo) [reply Conch/conch.conf.dist]: Done.

**Pull request merged**